Marlin/src/HAL/HAL_STM32F1/HAL.cpp

```diff
@@ -131,9 +131,15 @@ uint8_t adc_pins[] = {
   #if ENABLED(FILAMENT_WIDTH_SENSOR)
     FILWIDTH_PIN,
   #endif
+  #if ENABLED(POWER_MONITOR_CURRENT)
+    POWER_MONITOR_CURRENT_PIN,
+  #endif
+  #if ENABLED(POWER_MONITOR_VOLTAGE)
+    POWER_MONITOR_VOLTAGE_PIN,
+  #endif
 };
 
-enum TEMP_PINS : char {
+enum TempPinIndex : char {
```

Comment:
your renamed the enum type
need to rename the other one at line 290 :   TEMP_PINS pin_index;

Reply:
or restore TEMP_PINS

Reply:
yes I spotted it just after I asked. Thank you though @tpruvot :)